examples/api_gateway/api_gateway.h

```diff
@@ -166,6 +169,9 @@ struct onvm_parser_ipv4_5tuple {
  */
 struct data {
```

Comment:
Can you add the rx_fd to this struct and assign it in `dequeue_and_free_buffer_map`?